src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -96,15 +97,45 @@ public function iProceedSelectingOfflinePaymentMethod($paymentMethodName)
             'phoneNumber' => '321123456',
         ];
         $checkoutAddressingPage->fillAddressingDetails($addressingDetails);
-        $checkoutAddressingPage->pressButton('Continue');
+        $checkoutAddressingPage->forward();
 
         $checkoutShippingPage = $this->getPage('Checkout\CheckoutShippingStep');
-        $checkoutShippingPage->pressRadio('Free');
-        $checkoutShippingPage->pressButton('Continue');
+        $checkoutShippingPage->selectShippingMethod('Free');
 
         $checkoutPaymentPage = $this->getPage('Checkout\CheckoutPaymentStep');
-        $checkoutPaymentPage->pressRadio($paymentMethodName);
-        $checkoutPaymentPage->pressButton('Continue');
+        $checkoutPaymentPage->selectPaymentMethod($paymentMethodName);
+    }
+
+    /**
+     * @When /^I proceed selecting "([^"]+)" shipping method$/
+     * @Given /^I chose "([^"]*)" shipping method$/
```

Comment:
You mean here on in the scenario? Concept was, that ``I chose "..." shipping method`` has the same implementation as ``I proceed selecting "..." shipping method``, but feels more natural in case, when we change it in the next ``I change shipping method to "..."`` step.